Sources/NIOOpenSSL/SSLPKCS12Bundle.swift

```diff
@@ -0,0 +1,218 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftNIO open source project
+//
+// Copyright (c) 2017-2018 Apple Inc. and the SwiftNIO project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftNIO project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import CNIOOpenSSL
+import NIO
+
+
+/// A container of a single PKCS#12 bundle.
+///
+/// PKCS#12 is a specification that defines an archive format for storing multiple
+/// cryptographic objects together in one file. Its most common usage, and the one
+/// that SwiftNIO is most interested in, is its use to bundle one or more X.509
+/// certificates (`OpenSSLCertificate`) together with an associated private key
+/// (`OpenSSLPrivateKey`).
+///
+/// ### Working with TLSConfiguration
+///
+/// In many cases users will want to configure a `TLSConfiguration` with the data
+/// from a PKCS#12 bundle. This object assists in unpacking that bundle into its
+/// associated pieces.
+///
+/// If you have a PKCS12 bundle, you configure a `TLSConfiguration` like this:
+///
+///    let p12Bundle = OpenSSLPKCS12Bundle(file: pathToMyP12)
```

Comment:
I think there's one space missing here, right? Needs 5 in total, one to separate it from `///` and then four for code indent I think